vcf/parser.py

```diff
@@ -654,30 +651,30 @@ def __init__(self, stream, template, lineterminator="\n"):
         # get a maximum key).
         self.info_order = collections.defaultdict(
             lambda: len(template.infos),
-            dict(zip(template.infos.iterkeys(), itertools.count())))
+            dict(list(zip(iter(list(template.infos.keys())), itertools.count()))))
 
         two = '##{key}=<ID={0},Description="{1}">\n'
         four = '##{key}=<ID={0},Number={num},Type={2},Description="{3}">\n'
         _num = self._fix_field_count
-        for (key, vals) in template.metadata.iteritems():
+        for (key, vals) in list(template.metadata.items()):
```

Comment:
Call to list is probably superfluous. I see no side-effects to the metadata, so there's no reason to conjure a structure before iterating. I think the `list` is a context-free rewrite in case someone was storing the result rather than just immediately iterating on it.
```suggestion
        for (key, vals) in template.metadata.items():
```